Snippets/Snippets_6/UnitTesting/AdditionalServices/MyMessage.cs

```diff
@@ -0,0 +1,6 @@
+namespace Snippets6.UnitTesting.AdditionalServices
+{
+    class MyMessage
+    {
+    }
+}
```

Comment:
@timbussmann I could, but they are empty and provide no additional value, so I think it would just create noise